keps/sig-api-machinery/1164-remove-selflink/README.md

```diff
@@ -176,6 +177,22 @@ require more explanation.
 
 [deprecation-policy]: https://kubernetes.io/docs/reference/using-api/deprecation-policy/
 
+#### GA
+
+- Feature remained disabled by default for 4 releases
+- Feature disabled by default in cloud-providers: GKE, OpenShift
+- Sweep of RemoveSelfLink in [grep app](https://grep.app)
+  - The following issues were opened for repos explicitly setting `RemoveSelfLink=false`:
```

Comment:
Distro specific links is kind of what I expect, but interesting that people are disbaling the gate early.  Seems like people going through and explicitly trying to adopt perf improvement feature gates aggressively?  Would be good to think through general feature gate usage in distros and consumers and whether people are hurting themselves a bit.

Reply:
this was a deprecation gate, so disabling it *re-enables* the self link